ilastik/app.py

```diff
@@ -282,6 +284,12 @@ def logged_start(self):
         threading.Thread.start = logged_start
 
 
+def _init_preferences():
+    from volumina.utility.preferences import migrate
```

Comment:
i think it's fine to trigger it here. Could you maybe add a comment so that it's clear that it is some legacy pickle->json migration that is going on here.

Reply:
Maybe add the describing docstring to `migrate` instead?

Reply:
okay, let's do it this way